backend/YouTubeV2.Application/Services/SubscriptionService.cs

```diff
@@ -61,5 +61,32 @@ public async Task DeleteSubscriptionAsync(string subscribeeId, string subscriber
             _context.Subscriptions.Remove(subscription);
             await _context.SaveChangesAsync(cancellationToken);
         }
+
+        public async Task DeleteAllSubscriptionsRelatedToUserAsync(User user, CancellationToken cancellationToken = default)
+        {
+            var subscriptions = await _context.Subscriptions.Where(
+                subscription => subscription.Subscriber == user || subscription.Subscribee == user).ToListAsync(cancellationToken);
+
+            if (subscriptions == null || subscriptions.Count == 0)
+                return;
+
+            foreach (var subscription in subscriptions)
```

Comment:
there is method in ef RemoveRange or something like that idk how it works but probably we should use it